app/views/accessibility.njk

```diff
@@ -0,0 +1,76 @@
+{% extends './layouts/layout.njk' %}
+{% block pageTitle %}
+  Accessibility statement - {{siteTitle}}
+{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l">Accessibility statement for [website name] service</h1>
```

Comment:
I'd be inclined to use `serviceName` from `config.js` in place of the place holder.